lib/routes/api/login.js

```diff
@@ -31,8 +31,28 @@ router.post('/', async(req, res) => {
       return res.sendStatus(403);
     }
     logger.info({r}, 'successfully retrieved user account');
+
+    const maxLogginAttempts = process.env.LOGIN_ATTEMPTS_MAX_RETRIES || 6;
```

Comment:
typo can you change var name to `maxLoginAttempts`

Reply:
fixed